### PR TITLE
Clean up `pom.xml` and `Dockerfile`

### PR DESCRIPTION
**Pull request by dmytro-lazebnyi**

### CODE REVIEW

File: scalecube-configuration/pom.xml [hunk 1]
```diff
@@ -11,12 +13,11 @@
   <packaging>jar</packaging>
 
   <name>scalecube-configuration</name>
-  <url>http://maven.apache.org</url>
 
   <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <serviceName>${project.artifactId}</serviceName>
     <mainClass>io.scalecube.server.ConfigurationServiceRunner</mainClass>
+    <extratags>${env.TRAVIS_COMMIT}</extratags>
```
Comment (aharonha) [inline scalecube-configuration/pom.xml]: the same as https://github.com/scalecube/scalecube-organization-service/pull/58#discussion_r244578246

**Review by aharonha — DISMISSED** [review]
fix the extra tags if needed

**Review by segabriel — DISMISSED** [review]

**Review by aharonha — APPROVED** [review]

File: scalecube-configuration/pom.xml (hunk 1 above)
Comment (aharonha) [inline scalecube-configuration/pom.xml]:
```suggestion
    <!-- <extratags>${env.TRAVIS_COMMIT}</extratags> -->
```

File: pom.xml [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
-  <version>2.0.17-SNAPSHOT</version>
-
   <parent>
     <groupId>io.scalecube</groupId>
     <artifactId>scalecube-parent-pom</artifactId>
     <version>0.0.17</version>
   </parent>
+
+  <artifactId>scalecube-configuration-parent</artifactId>
+  <version>2.0.15-SNAPSHOT</version>
```
Comment (aharonha) [inline pom.xml]:
```suggestion
  <version>2.0.17-SNAPSHOT</version>
```

**Review by aharonha — CHANGES_REQUESTED** [review]
fix version

**Review by aharonha — APPROVED** [review]

**Review by segabriel — APPROVED** [review]

**Pull request merged**